src/components/common/MessageBox.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-card
+    outlined
+    class="rounded-0 ma-0"
+    :class="{
+      'message-box-gold': color === 'gold',
+      'message-box-red': color === 'red'
+    }"
+  >
+    <div class="mx-7 my-5 pa-0">
+      <slot />
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+@Component({})
+export default class MessageBox extends Vue {
+  @Prop({ required: true }) readonly color!: 'gold' | 'red'
```

Comment:
This doesn't actually display an error if the parent component uses a different string value, so for now it's "documentation" for a developer that uses this component.